tests/llmcompressor/pytorch/modifiers/smoothquant/test_pytorch.py

```diff
@@ -8,22 +6,22 @@
 from tests.llmcompressor.pytorch.helpers import LinearNet
 
 
-@pytest.mark.unit
-class TestSmoothQuantMapping(unittest.TestCase):
-    def setUp(self):
-        self.model = LinearNet()
-        self.state = State(model=self.model)
+@pytest.fixture
```

Comment:
nit: this seems to be identically defined in at least two places, if there were more than two then it might be better to hoist this to somewhere it can be imported or to `conftest` where it because automatically available without needing import.